src/Orleans.Core.Abstractions/IDs/GrainId.cs

```diff
@@ -292,6 +292,15 @@ internal string ToParsableString()
             return Key.ToHexString();
         }
 
+        /// <summary>
+        /// Return this GrainId ina  standard components form, suitable for later use with the <see cref="FromKeyInfo"/> method.
```

Comment:
Nit: a typo: "ina".

Reply:
Nice. There is also a double space after `ina` 😛. Thanks for finding these 🙂.